stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1363,6 +1370,9 @@ public enum TestRunPredicate : CustomStringConvertible {
     case linuxAny(reason: let reason):
       return "linuxAny(*, reason: \(reason))"
 
+    case androidAny(reason: let reason):
+      return "AndroidAny(*, reason: \(reason))"
```

Comment:
Any reason the string returned here is upper-cased and the two around it are lower-cased?